server.js

```diff
@@ -1,8 +1,19 @@
 const cron = require('node-cron');
+const Agenda = require('./app/modules/jobs/agenda');
+// const Agenda = require('agenda')
+// const moogoose = require('mongoose');
+// moogoose.connect('mongodb://localhost/TicketManagement')
+//     .then(() => { console.log("connected to mongodb") })
+//     .catch(err => {
+//         console.log("error in connecting to db")
+//     })
 const viewCurrentBookingsQuery = require('./cron/show_time')
     //const sendNotification = require('./cron/show_time')
 const dotenv = require('dotenv');
 var moment = require('moment');
+const mongoConnectionString = 'mongodb://127.0.0.1/agenda';
+
+//const agenda = new Agenda({ db: { address: mongoConnectionString, collection: 'Bookings' } });
 moment().format();
```

Comment:
What is this?

Reply:
As per the docs, giving connection and creating object .